apps/src/levelbuilder/skills/SkillsApi.tsx

```diff
@@ -0,0 +1,76 @@
+import MetricsReporter from '@cdo/apps/metrics/MetricsReporter';
+import {MetricEvent} from '@cdo/apps/metrics/events';
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+
+interface Skill {
+  key: string;
+  description: string;
+  evaluationCriteria: string;
+  concept: string;
+}
+
+export async function createSkill(skill: Skill) {
+  try {
+    const response = await fetch(`/skills`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': await getAuthenticityToken(),
+      },
+      body: JSON.stringify(skill),
+    });
+    if (!response.ok) {
+      const errorData = await response.json();
+      const msg = `Network response was not ok creating skill ${JSON.stringify(
+        errorData,
+        null,
+        2
+      )}`;
+      console.error(msg);
+      return null;
+    }
+    const data = await response.json();
```

Comment:
optional nit: Because we have a series of `await`, I tend to prefer the Promise syntax of `promise.then().then().catch()`. Just personal preference though.